### PR TITLE
Comments out code linking to the work page from the rest of 360 in pr…

### PR DESCRIPTION
**Pull request by Jacob-Bradley**
…eperation for going live. Also removes code from the Hello World which is no longer necessary.

### CODE REVIEW

**Review by adamprinciotta — COMMENTED** [review]
On the commented out timesheetButton, I'd add a comment saying to re-enable it when we have it fully working as we want it

File: src/components/Nav/components/NavLinks/index.js [hunk 1]
```diff
@@ -242,16 +244,16 @@ export default class GordonNavLinks extends Component {
...
-        <NavLink className="gc360-link" exact to="/timesheets" onClick={this.props.onLinkClick}>
-          <ListItem button>
-            <ListItemIcon>
-              <WorkIcon />
-            </ListItemIcon>
-            <ListItemText primary="Timesheets" />
-          </ListItem>
-        </NavLink>
-      );
+      //timesheetButton = (
```
Comment (NathanielRudenberg) [inline src/components/Nav/components/NavLinks/index.js:247]: I would add a comment here that suggests re-enabling when it's ready

**Review by NathanielRudenberg — CHANGES_REQUESTED** [review]
I would add a comment that suggests re-enabling it when the feature is ready for the public

**Comment by Jacob-Bradley** [comment]
Added requested change.

File: src/components/Nav/components/NavLinks/index.js [hunk 2]
```diff
@@ -242,7 +242,7 @@ export default class GordonNavLinks extends Component {
           </ListItem>
         </NavLink>
       );
-
+      // Re-Enable when adding the link to the timesheet page
```
Comment (NathanielRudenberg) [inline src/components/Nav/components/NavLinks/index.js:245]: Das ist gut

LGTM

**Review by NathanielRudenberg — APPROVED** [review]
LGTM

**Pull request merged**